pandas/tseries/converter.py

```diff
@@ -172,7 +172,7 @@ def _dt_to_float_ordinal(dt):
     is a :func:`float`.
     """
     if (isinstance(dt, (np.ndarray, Index, Series)
-                   ) and is_datetime64_ns_dtype(dt)):
```

Comment:
We could maybe add a test for this (and for tz aware Timestamp): https://github.com/pandas-dev/pandas/blob/master/pandas/tseries/tests/test_converter.py#L58